analytics/Pipfile

```diff
@@ -8,7 +8,7 @@ verify_ssl = true
 [packages]
 falcon = "*"
 sqlalchemy = "*"
-psycopg2 = "*"
+psycopg2-binary = "*"
```

Comment:
I think I've seen @AetherUnbound switch this from one to the other before... but I can't remember which direction 🤔 Just want to confirm that we're converging on a single solution for it 🙂

Reply:
I was making our `Pipfiles` consistent and settled on `pyscopg2-binary` in all of them because, unlike regular `psycopg2`, it doesn't need the PostgreSQL dev packages installed to compile. This was just easier 😆.

Reply:
This would be a regression as I mentioned in #182 the binary package is discouraged in production. [^1] The source package is preferable for the ingestion_server and the analytics projects too.

[^1]: [Psycopg documentation > Installation > psycopg vs psycopg-binary](https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary)

Reply:
Thanks @krysal! I switched to all `Pipfiles` to regular `psycopg2` in 0f4039b, and it reduced several MBs in all the images. So that's an absolute win!